Library/Homebrew/tap.rb

```diff
@@ -536,15 +535,33 @@ def formula_renames
 
   # Hash with tap migrations
   def tap_migrations
-    require "json"
-
     @tap_migrations ||= if (migration_file = path/"tap_migrations.json").file?
       JSON.parse(migration_file.read)
     else
       {}
     end
   end
 
+  # Hash with audit exceptions
+  def audit_exceptions
+    @audit_exceptions = {}
+
+    Dir[path/"audit_exceptions/*"].each do |exception_file|
```

Comment:
Use `Pathname.glob` then you can use `exception_file.basename` and `exception_file.read`